discord/iterators.py

```diff
@@ -501,46 +499,28 @@ def __init__(
 
         self.entries = asyncio.Queue()
 
-        if self.reverse:
-            self._strategy = self._after_strategy
-            if self.before:
-                self._filter = lambda m: int(m["id"]) < self.before.id
-        else:
-            self._strategy = self._before_strategy
-            if self.after and self.after != OLDEST_OBJECT:
-                self._filter = lambda m: int(m["id"]) > self.after.id
+        self._strategy = self._strategy_exec
 
-    async def _before_strategy(self, retrieve):
+    async def _strategy_exec(self, retrieve):
```

Comment:
I believe this is what doruk meant:
```suggestion

    async def _strategy(self, retrieve):
```

Reply:
That doesn't exist anymore